editor/scene_tree_editor.cpp

```diff
@@ -777,6 +777,8 @@ void SceneTreeEditor::_renamed() {
 		return;
 	}
 
+	new_name = new_name.strip_edges();
```

Comment:
```suggestion
    // Trim leading/trailing whitespace to prevent node names from containing accidental whitespace, which would make it impossible to get the node via `get_node()`.
	new_name = new_name.strip_edges();
```

Reply:
Thanks.

Reply:
I used githubs "Commit suggestion" button, which created a new commit. I think I need to "squash" these commits now, right? I'll look into it.